editor/export_template_manager.cpp

```diff
@@ -195,7 +195,7 @@ void ExportTemplateManager::_install_from_file(const String &p_file) {
 			data.resize(info.uncompressed_size);
 
 			//read
-			ret = unzOpenCurrentFile(pkg);
+			unzOpenCurrentFile(pkg);
```

Comment:
That's right. There's a reason why it's configurable. I mostly pay attention to this kind of `cppcheck` message only because it sometimes help bring some possible bugs to the surface like the one reduz pointed out. I don't mind undoing any of these changes at all if they are unwanted.